doc-content/enterprise-only/pages/building-custom-dashboard-widgets-creating-dashboard-proc.adoc

```diff
@@ -8,7 +8,7 @@ There are four stages in the creation of a custom dashboard:
 * Data set authoring - In this stage, you define a data set for accessing the data and displaying it through the pages. For more information, see xref:data_sets_add_proc_building-custom-dashboard-widgets[Adding data sets].
 * Page authoring - In this stage, you create the pages which are used to display the data from the data sets, among other things. For more information, see xref:building-custom-dashboard-widgets-creating-pages-proc[Creating pages].
 * Publication - In this stage, navigation between pages are defined when you create the custom navigation trees or modify the existing default one (*Workbench* tree). For more information, see xref:building-custom-dashboard-widgets-creating-navigation-tree-proc[Creating a navigation tree] or xref:building-custom-dashboard-widgets-editing-navigation-tree-con[Editing a navigation tree].
-* Security management - In this stage, role and group permissions are set which defines the privileges a user enjoys when working on {CENTRAL}. For more information, see xref:business-central-settings-security-proc_building-custom-dashboard-widgets[Security management].
```

Comment:
Consider changing the word "enjoys" to "in granted"

Reply:
Done